### PR TITLE
owo whats this? *sips tea*

### PR DESCRIPTION
**Pull request by optimumtact**

### CODE REVIEW

**Comment by SpaceJohn413** [comment]
why tho

**Comment by CosmicScientist** [comment]
ARE YE CHECKING ME BOTTOM LAD? I'LL GLASS YOU UP GUV GOOD AND PROPER WITH A CHESHIRE SMILE I WILL!

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -62,7 +62,8 @@
         "@pick(bug)",
         "@pick(semicolon)wtf??????????? @pick(bug)",
         "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
-        "DON'T EVER TUCH ME"
+        "DON'T EVER TUCH ME",
+        "OOH WOOOOOH WOTS THISSE MATE IS DAT YA KNOB OOO",
```
Comment (ShizCalev) [inline strings/brain_damage_lines.json]: remove the extra comma.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by PKPenguin321** [comment]
https://www.youtube.com/watch?v=vRUkesSLcAI

**Comment by JJRcop** [comment]
@lzimann

**Pull request closed without merging**